simd/unit_tests/include/TestSIMD_Construction.hpp

```diff
@@ -82,12 +82,38 @@ inline void host_test_simd_alias() {
   }
 }
 
+template <typename Abi, typename DataType>
```

Comment:
`Abi` is not used.
Either drop it or do
```suggestion
template <typename /*Abi*/, typename DataType>
```